src/config.c

```diff
@@ -89,156 +96,129 @@ int kBudSSLClientIndex = -1;
 int kBudSSLSNIIndex = -1;
 static const int kBudDefaultKeepalive = 3600;
 static const int kBudBackendMapSize = 1024;
+static const int kBudFileCacheSize = 64;
+
+static const char* bud_long_flags = "vi:c:dp";
+static struct option bud_long_options[] = {
```

Comment:
No biggie I thought so as well. I was just suggesting it because I've encountered a couple of flag change trips in personal projects.